### PR TITLE
Consume terra dev site 1.0

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
Consumes latest terra dev site

Files have been moved to protect the innocent.

Deployment Link:
https://terra-framework-deploye-pr-183.herokuapp.com/#/components

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-application-header-layout/src/terra-dev-site/doc/ApplicationHeader.doc.jsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 // Component Source
-import HeaderLayoutSrc from '!raw-loader!../src/ApplicationHeaderLayout.jsx';
+import HeaderLayoutSrc from '!raw-loader!../../../src/ApplicationHeaderLayout.jsx';
 
 // Example File
-import HeaderWireframe from './index-examples/HeaderWireframe';
-import HeaderWireframeSrc from '!raw-loader!./index-examples/HeaderWireframe.jsx';
+import HeaderWireframe from './example/HeaderWireframe';
+import HeaderWireframeSrc from '!raw-loader!./example/HeaderWireframe.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-header-layout/src/terra-dev-site/doc/ApplicationHeader.doc.jsx]: This needs to come from src

File: packages/terra-application-layout/src/terra-dev-site/doc/application-layout/Example.doc.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 import React from 'react';
 import DocTemplate from 'terra-doc-template';
-import { name } from '../package.json';
-import ExampleDoc from '../docs/example.md';
-import ExampleApplication from './index-examples/ExampleApplication';
-import ExampleApplicationSrc from '!raw-loader!./index-examples/ExampleApplication.jsx';
+import { name } from '../../../../package.json';
+import ExampleDoc from '../../../../docs/example.md';
+import ExampleApplication from '../example/ExampleApplication';
+import ExampleApplicationSrc from '!raw-loader!../example/ExampleApplication.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-layout/src/terra-dev-site/doc/application-layout/Example.doc.jsx]: This needs to be imported from src

File: packages/terra-application-links/src/terra-dev-site/doc/ApplicationLinks.doc.jsx [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
-import ApplicationLinkConfig from './index-examples/ApplicationLinkConfig';
+import { name } from '../../../package.json';
+import ReadMe from '../../../docs/README.md';
+import ApplicationLinkConfig from './common/ApplicationLinkConfig';
 
 // Component Source
-import TabsSrc from '!raw-loader!../src/tabs/ApplicationTabs.jsx';
-import TabExampleSrc from '!raw-loader!./index-examples/ApplicationTabsExample.jsx';
+import TabsSrc from '!raw-loader!../../../src/tabs/ApplicationTabs.jsx';
+import TabExampleSrc from '!raw-loader!./example/ApplicationTabsExample.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-links/src/terra-dev-site/doc/ApplicationLinks.doc.jsx]: This needs to be imported from src

File: packages/terra-application-menu-layout/src/terra-dev-site/doc/ApplicationMenuLayout.doc.jsx [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 
 // Component Source
-import MenuLayoutSrc from '!raw-loader!../src/ApplicationMenuLayout.jsx';
+import MenuLayoutSrc from '!raw-loader!../../../src/ApplicationMenuLayout.jsx';
 
 // Example File
-import MenuWireframe from './index-examples/MenuWireframe';
-import MenuWireframeSrc from '!raw-loader!./index-examples/MenuWireframe.jsx';
+import MenuWireframe from './example/MenuWireframe';
+import MenuWireframeSrc from '!raw-loader!./example/MenuWireframe.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-menu-layout/src/terra-dev-site/doc/ApplicationMenuLayout.doc.jsx]: this needs to be imported from src

File: packages/terra-application-name/src/terra-dev-site/doc/ApplicationName.doc.jsx [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
-import ReadMe from '../docs/README.md';
+import { name } from '../../../package.json';
+import ReadMe from '../../../docs/README.md';
 
 // Component Source
-import ApplicationHeaderNameSrc from '!raw-loader!../src/ApplicationHeaderName.jsx';
-import ApplicationHeaderNameStandardSrc from '!raw-loader!./index-examples/ApplicationHeaderNameStandard.jsx';
-import ApplicationMenuNameStandardSrc from '!raw-loader!./index-examples/ApplicationMenuNameStandard.jsx';
+import ApplicationHeaderNameSrc from '!raw-loader!../../../src/ApplicationHeaderName.jsx';
+import ApplicationHeaderNameStandardSrc from '!raw-loader!./example/ApplicationHeaderNameStandard.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-name/src/terra-dev-site/doc/ApplicationName.doc.jsx]: this needs to be imported from src

File: packages/terra-application-name/src/terra-dev-site/doc/ApplicationName.doc.jsx [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+import ApplicationMenuNameStandardSrc from '!raw-loader!./example/ApplicationMenuNameStandard.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-name/src/terra-dev-site/doc/ApplicationName.doc.jsx]: this needs to be imported from src

File: packages/terra-application-utility/src/terra-dev-site/doc/ApplicationUtility.doc.jsx [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
-import ApplicationHeaderUtilityExample from './index-examples/ApplicationHeaderUtilityExample';
-import ApplicationMenuUtilityExample from './index-examples/ApplicationMenuUtilityExample';
-import MenuUtilityMenuExample from './index-examples/MenuUtilityMenuExample';
-import ApplicationHeaderUtilityExampleSrc from '!raw-loader!./index-examples/ApplicationHeaderUtilityExample.jsx';
-import MenuUtilityMenuExampleSrc from '!raw-loader!./index-examples/MenuUtilityMenuExample.jsx';
-import ApplicationMenuUtilityExampleSrc from '!raw-loader!./index-examples/ApplicationMenuUtilityExample.jsx';
+import ApplicationHeaderUtilityExample from './example/ApplicationHeaderUtilityExample';
+import ApplicationMenuUtilityExample from './example/ApplicationMenuUtilityExample';
+import MenuUtilityMenuExample from './example/MenuUtilityMenuExample';
+import ApplicationHeaderUtilityExampleSrc from '!raw-loader!./example/ApplicationHeaderUtilityExample.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-utility/src/terra-dev-site/doc/ApplicationUtility.doc.jsx]: src

File: packages/terra-application-utility/src/terra-dev-site/doc/ApplicationUtility.doc.jsx [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
+import MenuUtilityMenuExampleSrc from '!raw-loader!./example/MenuUtilityMenuExample.jsx';
+import ApplicationMenuUtilityExampleSrc from '!raw-loader!./example/ApplicationMenuUtilityExample.jsx';
```
Comment (mjhenkes) [inline packages/terra-application-utility/src/terra-dev-site/doc/ApplicationUtility.doc.jsx]: you get the idea... We should brainstorm and see if we can figure out a easier answer than all this ../ stuff

File: packages/terra-application-header-layout/src/terra-dev-site/doc/ApplicationHeader.doc.jsx (hunk 1 above)
Comment (mjhenkes) [reply packages/terra-application-header-layout/src/terra-dev-site/doc/ApplicationHeader.doc.jsx]: This should have failed to compile, but there is a bug in my hot reloading logic.

File: packages/terra-application-utility/src/terra-dev-site/doc/ApplicationUtility.doc.jsx (hunk 8 above)
Comment (mjhenkes) [reply packages/terra-application-utility/src/terra-dev-site/doc/ApplicationUtility.doc.jsx]: Some ideas. I'm not sure I like any of them.
https://gist.github.com/branneman/8048520

File: jestconfig.json [hunk 9]
```diff
@@ -3,6 +3,9 @@
     "raf/polyfill",
     "./jestsetup.js"
   ],
+  "testMatch": [
```
Comment (emilyrohrbough) [inline jestconfig.json]: Curious, we didn't update any of the jest tests, so why did we add this?

Comment (mjhenkes) [reply jestconfig.json]: Jest matching is pretty generous and was trying to match on our .test.jsx examples for terra-dev-site. this restricts jest matching to look for jest tests only in the the jest folder

Comment (emilyrohrbough) [reply jestconfig.json]: I think I must have asked this before because that response sounds similar.

**Comment by bjankord** [comment]
@mmalaker For functional review, let's spot check some of the [navigation for components](https://terra-framework-deploye-pr-183.herokuapp.com/#/components/terra-aggregator/aggregator) and [tests routes](https://terra-framework-deploye-pr-183.herokuapp.com/#/tests).

**Comment by mmalaker** [comment]
Functional spot check completed. The Terra Framework site displays and functions as expected.

**Pull request merged**